.github/workflows/test_build_deploy.yml

```diff
@@ -309,6 +223,7 @@ jobs:
         with:
           tag_name: ${{env.TAG}}
           name: ${{env.TAG}}
+          body_path: ./CHANGELOG.md
```

Comment:
This will add the changelog automatically to the github release

Reply:
is this not happening on web? it happens on tm but maybe its handled elsewhere I'm not sure now.

Reply:
yes this is included on TM, just checked.

Reply:
It was happening for production releases, but not staging for some reasons ... 🙃